public/sounds.json

```diff
@@ -203,5 +204,22 @@
         "to": 14.5
       }
     }
+  },
+  {
+    "//":"For translators: this is a meme of Watame, see https://www.youtube.com/watch?v=PJf3XZ636-0",
```

Comment:
Is it Watame-related? I think the comment can be removed if there's no special things about the translation.